src/artm_tests/collection_parser_test.cc

```diff
@@ -19,51 +19,11 @@ TEST(CollectionParser, UciBagOfWords) {
   ::artm::CollectionParserConfig config;
   config.set_format(::artm::CollectionParserConfig_Format_BagOfWordsUci);
   config.set_target_folder(target_folder);
-  config.set_dictionary_file_name("test_parser.dictionary");
-  config.set_gather_cooc(true);
-  config.add_cooccurrence_token("token1");
-  config.add_cooccurrence_token("token2");
-  config.add_cooccurrence_token("token3");
   config.set_num_items_per_batch(1);
   config.set_vocab_file_path("../../../test_data/vocab.parser_test.txt");
   config.set_docword_file_path("../../../test_data/docword.parser_test.txt");
 
-  std::shared_ptr< ::artm::DictionaryConfig> dictionary_parsed = ::artm::ParseCollection(config);
```

Comment:
Could you turn this test into a test of ArtmGatherDictionary? To do this you also need a method `ArtmRequestDictionary` that would return a `DictionaryData` message. I think it would be greate to do this as part of this pull request.